src/Sentry/Internal/ReplaySession.cs

```diff
@@ -0,0 +1,33 @@
+#if __ANDROID__
+using Sentry.Android.Extensions;
+#endif
+
+namespace Sentry.Internal;
+
+internal interface IReplaySession
+{
+    public SentryId? ActiveReplayId { get; }
+}
+
+internal class ReplaySession : IReplaySession
+{
+    public static readonly IReplaySession Instance = new ReplaySession();
+
+    private ReplaySession()
+    {
+    }
+
+    public SentryId? ActiveReplayId
+    {
+        get
+        {
+#if __ANDROID__
+            // Check to see if a Replay ID is available
+            var replayId = JavaSdk.ScopesAdapter.Instance?.Options?.ReplayController?.ReplayId?.ToSentryId();
+            return (replayId is { } id && id != SentryId.Empty) ? id : null;
+#else
+            return null;
```

Comment:
Is the idea to add iOS here later? potentially Web for example if we were brave enough?

Reply:
Yeah, I was planning for at least iOS...